libs/designer/src/lib/core/state/designerOptions/designerOptionsSlice.ts

```diff
@@ -111,6 +111,7 @@ export const designerOptionsSlice = createSlice({
         ...state.hostOptions,
         ...action.payload.hostOptions,
       };
+      state.designerOptionsInitialized = true;
```

Comment:
just checking, should this be "state.designerOptions.designerOptionsInitialized" ?

Reply:
Also, I am guessing this is the default value? It should be "false" right?

Reply:
Since we're in the designerOptions slice, the `state` variable here is already in the `designerOptions` section of the state, so just `state.designerOptionsInitialized` is correct
And the default value for this is just undefined but I can make it explicit as false as the default state.  This call should be true since after this is run it shows that we initialized the designer options state.